Modules/Core/Common/include/itkAnatomicalOrientation.h

```diff
@@ -0,0 +1,543 @@
+/*=========================================================================
+ *
+ *  Copyright NumFOCUS
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         https://www.apache.org/licenses/LICENSE-2.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *=========================================================================*/
+#ifndef itkAnatomicalOrientation_h
+#define itkAnatomicalOrientation_h
+
+#include "ITKCommonExport.h"
+#include "itkImageBase.h"
+#ifndef ITK_FUTURE_LEGACY_REMOVE
+#  include "itkSpatialOrientation.h"
+#endif
+#include <map>
+#include <string>
+
+namespace itk
+{
+
+/** \class AnatomicalOrientation
+ * \brief A class to represent anatomical orientations and convert between conventions.
+ *
+ *  Defines patient specific anatomical names to the XYZ axes of a 3D image.
+ *
+ * \todo update the details of the enum names in the docs
+ *  Instances hold the patient orientation enum and allow conversion to and from enums, string and direction cosine
+ * matrices. Conversions from a direction cosine matrix is approximated with the orientation of the closes axes.
+ *
+ * \ingroup ITKCommon
+ */
+class ITKCommon_EXPORT AnatomicalOrientation
+{
+public:
+  static constexpr unsigned int Dimension = 3;
+  using DirectionType = typename ImageBase<Dimension>::DirectionType;
+  static constexpr unsigned int ImageDimension = Dimension;
+
+#ifndef ITK_FUTURE_LEGACY_REMOVE
+  using LegacyOrientationType = SpatialOrientationEnums::ValidCoordinateOrientations;
+#endif
+
+  enum class CoordinateEnum : uint8_t
+  {
+    UNKNOWN = 0,
+#ifdef ITK_AMBIGUOUS_TO_COORDINATE_ENUMS
+    Left = 2,
+    Right = 3,
+    Anterior = 4,
+    Posterior = 5,
+    Superior = 8,
+    Inferior = 9,
+#endif
```

Comment:
Are these enumerators meant for FUTURE_REMOVE?

Reply:
I don't particularly care. As noted by the conditional these terms are ambiguous, and the opposite of the the current SpatialOrientationEnums::[CoordinateTerms](https://itk.org/Doxygen/html/group__ITKCommon.html#gaeebc5f41f756b247ec3e5179b03102f9). The terms below like "RightToLeft" a clear an unambiguous and should be used.

Whatever is easiest for the community with these terms is likely best.

Reply:
I have removed the conditional.